tests/precommit/test.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        export REPO="$(pwd)/../.."
+        export REV="$(git rev-parse --short HEAD)"
```

Comment:
repo and rev are required options - this test _current_ HEAD so if you are going to play with pre-commit don't forget to commit local changes.